src/events/aws_sdk.js

```diff
@@ -546,6 +546,24 @@ const stepFunctionsEventCreator = {
 
             break;
         }
+        case 'sendTaskSuccess': {
```

Comment:
I didn't see a reason to catch sendTaskFailure, Because it's the last step of the machine, And I don't want to pass it with a new id and step_num=-1. 
![image](https://user-images.githubusercontent.com/17531869/74022686-bc410d00-49a6-11ea-91ee-a79e1adccc13.png)

Reply:
I think that it doesn't always have to be the last step (you can configure task transition after failure), but anyway I see that there isn't an `output` field to use